python/paddle/fluid/imperative/layer_object_helper.py

```diff
@@ -191,13 +191,7 @@ def append_activation(self,
             act['use_mkldnn'] = use_mkl_dnn
         act_type = act.pop('type')
 
-        tmp = input_var
-        # NOTE(dzhwinter): some activation support inplace compution.
-        # NOTE(minqiyang): currently, we don't support inplace in imperative mode
-        if not _in_imperative_mode() and core.IsInplace(act_type):
-            tmp = input_var
-        else:
-            tmp = self.create_variable_for_type_inference(dtype=input_var.dtype)
+        tmp = self.create_variable_for_type_inference(dtype=input_var.dtype)
```

Comment:
why change this? we don't support inplace for imperative mode? @velconia

Reply:
this logic is duplicate with the inplace_op_pass in the c++ side. And it may cause circle when used with inplace_op_pass

there 's also a related ticket complain add inplace logic here it's weird:
http://newicafe.baidu.com/issue/DLTP-448/show?from=page